### ISSUE TITLE
TFProf is unable to parse the profile generated by profileContext.

### ISSUE THREAD

**Issue by rahul003** [tensorflow/tensorflow#23231]
TFProf is unable to parse the profile generated by profileContext.

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): Yes
- OS Platform and Distribution : Linux Ubuntu 18.04: 
- TensorFlow installed from (source or binary): source
- TensorFlow version (use command below): 1.11
- Python version: 3.6
- Bazel version (if compiling from source): 0.16.1
- GCC/Compiler version (if compiling from source): 7.3
- CUDA/cuDNN version: 10.0
- GPU model and memory: V100, 16GB

**Describe the current behavior**
TFProf unable to parse the profile generated by profileContext. 

**Describe the expected behavior**
When I generate the profile file by using the low level approach of runOptions, runMetadata, and then writing the string generated by profiler.serialize_to_string(), I can load it with tfprof and analyze it just fine. This should happen even when using ProfileContext.

**Code to reproduce the issue**
```
with tf.contrib.tfprof.ProfileContext('./profiles', trace_steps=range(50,100), dump_steps=[100]) as pctx:

 ... Regular session.run

```

**Comment by rahul003** [comment]
I tried this twice, one where I killed the program after the trace was dumped. Other where I made the training loop exit cleanly. 
```
/home/ubuntu/tensorflow/bazel-bin/tensorflow/core/profiler/profiler
--profile_path=profiles/profile_100
Reading Files...
Failed to parse profile
tfprof>
Cannot start interative shell, use 'bazel-bin' instead of 'bazel run'.
(tf_p36_cu10) ubuntu@ip-172-31-43-37:~/OpenSeq2Seq$
```
```
(tf_p36_cu10) ubuntu@ip-172-31-43-37:~/OpenSeq2Seq$ tfprof --profile_path=profiles/profile_70
/home/ubuntu/tensorflow/bazel-bin/tensorflow/core/profiler/profiler
--profile_path=profiles/profile_70
Reading Files...
[libprotobuf ERROR external/protobuf_archive/src/google/protobuf/wire_format_lite.cc:626] String field 'tensorflow.tfprof.ProfileProto.IdToStringEntry.value' contains invalid UTF-8 data when parsing a protocol buffer. Use the 'bytes' type if you intend to send raw bytes.
Failed to parse profile
tfprof> op -select micros,bytes,occurrence -order_by micro
Cannot start interative shell, use 'bazel-bin' instead of 'bazel run'.
(tf_p36_cu10) ubuntu@ip-172-31-43-37:~/OpenSeq2Seq$ tfprof --profile_path=profiles/profile_100
```

**Comment by ChrisAntaki** [comment]
Wish I could help, maybe another TensorFlow contributor is more familiar with the C++ behind TFProf?

**Comment by shlens** [comment]
Wish I could help but I do not know much about the C++ interface.

**Comment by harshini-gadige** [comment]
@shlens @ChrisAntaki   -  Could you please suggest a right person to assign this issue ?

**Comment by tensorflowbutler** [comment]
Nagging TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by mohantym** [comment]
Hi @rahul003 !We see that you are using old version of Tensorflow which is officially considered as end of life, We recommend that you upgrade to 2.4 or later version and let us know if the issue still persists in newer versions .Please open a new issue in case you face any errors, we will get you the right help .Thanks!
